internal/terraform/node_action_trigger_plan.go

```diff
@@ -7,16 +7,19 @@ import (
 	"fmt"
 
 	"github.com/hashicorp/hcl/v2"
+	"github.com/zclconf/go-cty/cty"
+
 	"github.com/hashicorp/terraform/internal/addrs"
 	"github.com/hashicorp/terraform/internal/configs"
+	"github.com/hashicorp/terraform/internal/instances"
 	"github.com/hashicorp/terraform/internal/tfdiags"
 )
 
 type nodeActionTriggerPlanExpand struct {
-	actionAddress     addrs.ConfigAction
-	actionInstanceKey addrs.InstanceKey // TODO: This should probably be a new address? Look at resources
-	resolvedProvider  addrs.AbsProviderConfig
-	actionConfig      *configs.Action
+	Addr             addrs.ConfigAction
+	ActionExpr       hcl.Expression
```

Comment:
```suggestion
```


I think we don't set or use this one but only the one in the lifecycle block (which makes more sense since the CLI triggered actions won't have this reference)

Reply:
Good catch, thank you!